packages/python/plotly/plotly/express/_core.py

```diff
@@ -1981,14 +2013,14 @@ def post_agg(dframe: nw.LazyFrame, continuous_aggs, discrete_aggs) -> nw.LazyFra
         - discrete_aggs is either [args["color"], <rest_of_cols>] or [<rest_of cols>]
         """
         return dframe.with_columns(
-            **{c: nw.col(c) / nw.col(count_colname) for c in continuous_aggs},
+            **{col: nw.col(col) / nw.col(count_colname) for col in continuous_aggs},
```

Comment:
should this one also use `.alias`? (asking in case it was forgotten - I really like the other changes you've made 🙌 )

Reply:
I think the entire block could reduce to:

```py
    return dframe.with_columns(
            *[nw.col(continuous_aggs) / nw.col(count_colname)],
            *[(nw.when(nw.col(f"{col}{n_unique_token}") == 1)
                .then(nw.col(col))
                .otherwise(nw.lit("(?)")).alias(col))
                for col in discrete_aggs
            ]
        ).drop([f"{col}{n_unique_token}" for col in discrete_aggs])
```
but not sure if it is better for readability. WDYT?

Reply:
personally, I like that! let's see what @emilykl thinks